WcaOnRails/spec/lib/middlewares/warden_user_logger_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'middlewares/warden_user_logger'
+require 'ostruct'
+
+describe Middlewares::WardenUserLogger do
+  describe "call" do
+    let(:app) { double(:app) }
+    let(:logger) { double(:logger) }
+    subject { described_class.new(app, logger) }
+
+    before do
```

Comment:
Yeah, `before` defaults to `:each`/`:example`. I'll change it for the sake of consistency.